### ISSUE TITLE
Export to drawing edges

### ISSUE THREAD

**Issue by SkyeNygaard** [mbrlabs/Lorien#93]
Right now the export functionality essentially acts like a screenshot, only exporting what is visible on the window. I believe most users would prefer the default functionality to be to export the whole canvas, with the corners set to make sure nothing is cut off. They can then use a screenshot tool to take a picture of the visible space on the canvas if they want.

**Comment by mbrlabs** [comment]
Yeah, the export is currently very barebones and really just a screenshot. Ideally we would have 2 export modes:

1) Export the whole canvas as a `svg` which can be scaled indefinitly just like Lorien files and opened by almost all graphics programs. This would require digging into the format and write an exporter to convert from the internal Lorien format to svg. Seems doable and like a pretty neat feature.
2) Export parts of the visible screen by draging over the area and exporting it as png (think of it like the crop tool in Krita/Photoshop etc. but for exporting). From a UX/UI perspective this is a lot of work and not as useful as svg. I think Milton did it this way

**Comment by mbrlabs** [comment]
I implemented a basic svg exporter now (and removed the png exporter). Right now it exports the brush strokes with a uniform thickness and ignores pressure data / stroke width. Have to figure out how to do that, since svg does not support something like that out of the box.

If you want to check it out, you can download the latest build [here](https://github.com/mbrlabs/Lorien/actions/runs/2345101186) (scroll down to articfacts)

**Comment by tmathews** [comment]
For anyone else facing this challenge, you can convert using the imagemagick convert program. It does require the CLI, but it works really well. Perhaps the application could wrap these commands if it detects imagemagick is installed.
